blocks/init/src/Blocks/components/paragraph/components/paragraph-options.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
-import { __, sprintf } from '@wordpress/i18n';
-import { ColorPaletteCustom, checkAttr, getAttrKey, icons, getOption } from '@eightshift/frontend-libs/scripts';
-import { SelectControl, Icon, ToggleControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { ColorPaletteCustom, checkAttr, getAttrKey, icons, getOption, ComponentUseToggle, IconLabel, CustomSelect } from '@eightshift/frontend-libs/scripts';
```

Comment:
probably not needed comment but it would be cleaner if we would do it something like this

```suggestion
import { ColorPaletteCustom, ComponentUseToggle, IconLabel, CustomSelect, checkAttr, getAttrKey, icons, getOption } from '@eightshift/frontend-libs/scripts';
```